plugin_dockerbuild/config.json-template

```diff
@@ -21,7 +21,7 @@
 			"Options": ["rbind"]
 		},
 		{
-			"Description" : "Location to look for config file (/etc/docker-volume-vsphere.conf)",
+			"Description" : "Location to look for config file (/etc/BINARY.conf)",
```

Comment:
Hmmm I am not sure how to use golang template here.
We don't have any go code. This is where we build the docker plugins.
So it's the script world...
It might be an overkill to add and compile some go code here in order to replace texts only :)
Let me add the pattern thing instead.